Source/WebKit/UIProcess/API/ios/WKWebViewIOS.h

```diff
@@ -231,6 +231,10 @@ enum class TapHandlingResult : uint8_t;
 - (void)_updatePageLoadObserverState NS_DIRECT;
 #endif
 
+#if ENABLE(MODEL_PROCESS)
+- (void)willInvalidateDraggedModelWithContainerView:(UIView *)containerView;
```

Comment:
You might want to prefix this internal method with an underscore, since `WKWebView` is public (we aren't super consistent about this rule, but it's generally a good idea to underscore-prefix internal methods on API objects to avoid conflicting with clients' category methods)